packages/intrepid2/CMakeLists.txt

```diff
@@ -30,6 +30,13 @@ ELSE()
     OFF)
 ENDIF()   
 
+# temporary flag
+TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_KEEP_EXPERIMENTAL_NAMESPACE
```

Comment:
Can this be handled with the existing deprecation mechanisms?

Reply:
Well, we could, but I don't want to wait until the point release to remove the namespace. I just want to give Roger some time to make the changes in Empire.